packages/client/lib/net/peer/rlpxpeer.ts

```diff
@@ -1,16 +1,18 @@
-import { randomBytes } from 'crypto'
 import {
   Capabilities as Devp2pCapabilities,
   ETH as Devp2pETH,
   LES as Devp2pLES,
   Peer as Devp2pRlpxPeer,
   RLPx as Devp2pRLPx,
 } from '@ethereumjs/devp2p'
+import { isTruthy } from '@ethereumjs/util'
+// eslint-disable-next-line import/order
```

Comment:
I've noticed a couple of instances where the import/order rule was disabled with some imports. Why is that necessary?

Reply:
Cool thanks for the note!